tck/tck-runtime/src/main/java/servlet/tck/api/jakarta_servlet/dispatchtest/DispatchTests11.java

```diff
@@ -84,10 +80,12 @@ public void dispatchTest11(ServletRequest request, ServletResponse response)
     AsyncContext ac = request.startAsync();
     response.getWriter()
         .println("Before second dispatch=" + System.currentTimeMillis());
-    ac.dispatch(
-        request.getServletContext().getContext(DispatchTestServlet.getDispatcherContextRoot()),
-        "/DispatchTests?testname=dispatchTest");
+    ServletContext context = request.getServletContext().getContext(DispatchTestServlet.getDispatcherContextRoot());
+    if(context != null) {
```

Comment:
Why isn't this `if (context != null || ServletTestUtil.SUPPORT_CROSS_CONTEXT) ` ?
The same comment applies throughout this file.

Reply:
fixed with 7fcd003024438dc9abd49fe324e8d919fd227e22